### PR TITLE
feat: redo homepage MA-508

### PR DESCRIPTION
**Pull request by mihai-sysbio**
This PR aims to update the homepage of the website. It does not, however, update the contents of the news section.
There is also a refactoring commit https://github.com/MetabolicAtlas/MetabolicAtlas/pull/548/commits/a58dfdb8ce5945964b46fdeca81de2eaf01017a1, adopting the Bulma class `is-clickable` instead of the custom but identical `clickable`.
Moreover, the PR introduces UI improvements for the Explorer and Repository pages, where the models are now presented in more compact cards.

### CODE REVIEW

**Comment by mihai-sysbio** [comment]
Thanks @nanjiangshu and @e0 for the feedback:
- [ ] update comparison image
- [ ] rm link for community
- [ ] add explanation before version boxes
- [ ] increase size of images on mobile
- [ ] reduce font size for mobile
- [ ] change style from box to card for feature images
- [ ] remove card footer for GEM repository
- [ ] change "Latest news" to "What's new"
There is more feedback in a new card MA-933.

**Comment by mihai-sysbio** [comment]
This PR is ready for another look.

File: frontend/src/components/Home.vue [hunk 1]
```diff
@@ -141,186 +178,122 @@ export default {
...
-      menu: [
-        { title: 'Welcome',
-          text: '<b>Metabolic Atlas</b> integrates open source genome-scale metabolic models (GEMs) of human and yeast for easy browsing and analysis. It also contains many more GEMs constructed by our organization.<br><br>Detailed biochemical information is provided for individual model components, such as reactions, metabolites, and genes. These components are also associated with standard identifiers, facilitating integration with external databases, such as the Human Protein Atlas. <br><br><a href="https://doi.org/10.1126/scisignal.aaz1482" target="_blank">Robinson, J., et al, 2020. <i>An atlas of human metabolism</i>. Sci. Signal. 13, eaaz1482 </a>',
-          img: require('../assets/explorer.jpg'),
-          cardLink: 'Explore a GEM',
-          route: { name: 'explorer' },
-          icon: 'home' },
+      news: {
+        title: "What's new",
+        text: 'Here goes all the news, which ideally should link to the news page under About.',
```
Comment (e0) [inline frontend/src/components/Home.vue:183]: This looks like a placeholder text. Is there any Jira task in the current sprint to replace this?

File: frontend/src/components/Home.vue [hunk 2]
```diff
@@ -1,135 +1,172 @@
...
+                  Article under consideration.
+                </p>
+              </div>
+              <div class="box">
+                <p>Version 1.0: <a href="https://doi.org/10.1126/scisignal.aaz1482" target="_blank">Robinson, J., et al, 2020. <i>An atlas of human metabolism</i>. Sci. Signal. 13, eaaz1482</a></p>
+              </div>
+            </div>
+            <div class="column is-6 is-size-5">
+              <p v-html="news.text"></p>
+              <table>
```
Comment (e0) [inline frontend/src/components/Home.vue:65]: This table could use some spacing between the text.

File: frontend/src/components/Home.vue [hunk 3]
```diff
@@ -1,135 +1,172 @@
...
+                <template v-for="el in newsItems">
+                  <!-- eslint-disable-next-line vue/valid-v-for vue/require-v-for-key -->
+                  <tr v-if="el.date">
+                    <td>{{ el.date }}</td>
```
Comment (e0) [inline frontend/src/components/Home.vue:69]: I'm not familiar with using dot as a separator for dates, is this an academic thing?

File: frontend/src/components/Home.vue [hunk 4]
```diff
@@ -1,135 +1,172 @@
...
-                    <div class="card" :class="{ 'hoverable': currentMenu.cardLink}">
-                      <img :src="currentMenu.img" :alt="currentMenu.title" />
-                      <template v-if="currentMenu.cardLink">
-                        <footer class="card-footer has-text-centered has-background-primary-lighter">
-                          <a class="card-footer-item has-text-weight-bold">{{ currentMenu.cardLink }}</a>
-                        </footer>
-                      </template>
+                <div class="column is-offset-1-mobile is-10-mobile">
+                  <router-link :to="item.route">
+                    <div clas="card hoverable">
```
Comment (e0) [inline frontend/src/components/Home.vue:106]:
```suggestion
                    <div class="card hoverable">
```

File: frontend/src/components/Home.vue [hunk 5]
```diff
@@ -1,135 +1,172 @@
...
+              {{ news. title }}
+            </router-link>
+          </p>
+          <div class="columns is-variable is-8 pt-3">
+            <div class="column is-6 is-size-5">
+              <p>The main versions of <b>Metabolic Atlas</b> are associated with scientific articles as follows.</p><br>
+              <div class="box">
+                <p>Version 2.0:
+                  <i>Genome-scale reconstruction of metabolic networks of model animals
+                     represents a platform for translational research</i>.
```
Comment (e0) [inline frontend/src/components/Home.vue:55]:
```suggestion
                    represents a platform for translational research</i>.
```

Linter warning about indentation here.

**Review by e0 — CHANGES_REQUESTED** [review]
Got a few minor suggestions/comments but otherwise it looks good!

File: frontend/src/components/Home.vue (hunk 1 above)
Comment (mihai-sysbio) [reply frontend/src/components/Home.vue:183]: MA-664 should cover it.

File: frontend/src/components/Home.vue (hunk 3 above)
Comment (mihai-sysbio) [reply frontend/src/components/Home.vue:69]: I'm speaking from memory, but I think the date is used as a hash linking to the About page, so a the usual forward slash would be unusable. A dot felt also more readable than a `-`. What do you think?

File: frontend/src/components/Home.vue (hunk 4 above)
Comment (mihai-sysbio) [reply frontend/src/components/Home.vue]: Nice that you spotted this typo.

File: frontend/src/components/Repository.vue [hunk 6]
```diff
@@ -26,13 +26,9 @@
...
               </header>
-              <div class="card-content py-2 card-fullheight">
-                <p class="has-text-justified">{{ model.full_name }}</p><br>
-                <p>
-                  Reactions: {{ model.reaction_count }}<br>
-                  Metabolites: {{ model.metabolite_count }}<br>
-                  Genes: {{ model.gene_count }}<br><br>
-                  Updated {{ model.date || "n/a" }} from
+              <div class="card-content px-4 py-2 card-fullheight">
+                <p class="has-text-justified">
```
[the remark this answers is not in the document]
Comment (mihai-sysbio) [reply frontend/src/components/Repository.vue:30]: Yes!

File: frontend/src/components/Home.vue (hunk 3 above)
Comment (e0) [reply frontend/src/components/Home.vue:69]: I looked at https://en.wikipedia.org/wiki/Date_format_by_country and it looks like certain countries do use `.` but `-` seems to be more widely used. You can make the call here, I have a slight preference for `-` but it's not an important issue.

Comment (mihai-sysbio) [reply frontend/src/components/Home.vue:69]: Will fix as part of MA-664.

File: frontend/src/components/Home.vue (hunk 1 above)
Comment (mihai-sysbio) [reply frontend/src/components/Home.vue]: For now, resolved in 820e49d

File: frontend/src/components/Home.vue (hunk 2 above)
Comment (mihai-sysbio) [reply frontend/src/components/Home.vue:65]: Addressed through 4928dae

**Review by e0 — APPROVED** [review]

**Comment by mihai-sysbio** [comment]
Thanks for the detailed review @e0 and for the multiple rounds of thorough feedback.

**Pull request merged**